### PR TITLE
Feature#30 psycho edit page

### LINKED ISSUE #30: Psychologist edit page
# REQUIREMENTS
Psychologists can edit and save their profile
All fields can be edited
Responsive
Add styles to the Psychologist edit page

# DESIGN
Provide references to images of the design, how it looks now, and how it should look like.
image
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] The Psychologist can edit his profile.
- [ ] Responsive
- [ ] The Psychologists can save the changes.

### PR DESCRIPTION
**Pull request by xFerchoVJ**
# Description
- I've made the styles for the edit page (Mobile and Desktop)
- I've moved the functions in the controllers so that it could accept the fields that the user is going to edit.

Resolve #30 

  ## Screenshots 
#Desktop
image
#Mobile
image

### CODE REVIEW

File: app/controllers/psychologist/registrations_controller.rb [hunk 1]
```diff
@@ -7,13 +7,26 @@ class Psychologist::RegistrationsController < Devise::RegistrationsController
...
   end
+   
+  def update
+    super
+  end
+
+  protected
+
+  def after_update_path_for(profile_psychologist)
+    puts 'this is happening yoyo mama'
```
Comment (jclopezdev) [inline app/controllers/psychologist/registrations_controller.rb:18]: @xFerchoVJ  Do you really need this `puts` in your code?

Comment (xFerchoVJ) [reply app/controllers/psychologist/registrations_controller.rb:18]: Nooo
I forgot to delete that, sorry!

File: app/assets/stylesheets/home/components/_variables.scss [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+$white: #fff;
 $breakpoint-desktop: 992px;
 
 %flex-column{
   display: flex;
   flex-direction: column;
 }
 
-@mixin button($ptop, $pbot, $bgcolor, $color){
+@mixin button($ptop_bot, $pleft_right, $bgcolor, $color){
```
Comment (juliocabrera820) [inline app/assets/stylesheets/home/components/_variables.scss:10]: you may use a name more descriptive

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<footer class="text-muted text-center text-small profile_footer">
+  <p>
+  <%= link_to "Español", url_for(locale: :es) %> |
+  <%= link_to "English", url_for(locale: :en) %>
+  </p>
```
Comment (juliocabrera820) [inline app/views/layouts/_footer.html.erb:5]: add indentation for link_to helper

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -11,18 +11,17 @@
...
+      <%= render 'layouts/navbar' %>
+      <%= yield %>
+      <%= render 'layouts/footer' %>
+    <% else %>
+      <%= render 'layouts/navbar' %>
+      <%= yield %>
+      <%= render 'layouts/footer' %>
+      <p class="notice"> <%= notice %> </p>
+      <p class="alert"> <%= alert %> </p>
+    <% end %>
```
Comment (juliocabrera820) [inline app/views/layouts/application.html.erb:24]: this logic can be splitted into a partial as _flash_message

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+</footer>
```
Comment (YilKanda) [inline app/views/layouts/_footer.html.erb:6]: Add a line blank

**Review by YilKanda — COMMENTED** [review]
Check out the feedback the other reviewers have given you.

File: app/controllers/psychologist/registrations_controller.rb [hunk 6]
```diff
@@ -7,13 +7,26 @@ class Psychologist::RegistrationsController < Devise::RegistrationsController
...
     super
   end
+   
+  def update
+    super
+  end
+
+  protected
+
+  def after_update_path_for(profile_psychologist)
```
Comment (Ilyeo) [inline app/controllers/psychologist/registrations_controller.rb:17]: @xFerchoVJ  probably you could call this method using the `after_action` [Filters](https://guides.rubyonrails.org/action_controller_overview.html#filters)

**Review by juliocabrera820 — COMMENTED** [review]
check out the comments

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (xFerchoVJ) [reply app/views/layouts/application.html.erb:24]: We are going to add a task to improve flashes in another task!
In this case, just implement that part so the styles don't fail me.

**Review by AlejandroVI99 — APPROVED** [review]
LGTM

**Review by juliocabrera820 — APPROVED** [review]
Nice job ⚡

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**